tests/TestCase.php

```diff
@@ -85,12 +85,17 @@ public function getTempDirectory(): string
     protected function createTables(...$tableNames)
     {
         collect($tableNames)->each(function (string $tableName) {
-            $this->app['db']->connection()->getSchemaBuilder()->create($tableName, function (Blueprint $table) {
+            $this->app['db']->connection()->getSchemaBuilder()->create($tableName, function (Blueprint $table) use ($tableName){
                 $table->increments('id');
                 $table->string('name')->nullable();
                 $table->string('text')->nullable();
                 $table->timestamps();
                 $table->softDeletes();
+
+                if ($tableName === 'articles') {;
```

Comment:
Instead of putting an if here, just create a second  `$this->app['db']->connection()->getSchemaBuilder()->create` statement where you create the articles table.